
2022-10-25 18:17:48.558 - [Debug] [Connection] Attacker connected: 120.31.133.162 | Client Identification: SSH-2.0-Go
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
(node:327105) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'DATABASE_4',
  containerIp: '10.0.3.54',
  mitmPort: 1003,
  autoAccessFixed: 1
}
2022-10-25 18:18:48.656 - [Info] MITM Version: 2
2022-10-25 18:18:48.657 - [Info] Auto Access Enabled: true
2022-10-25 18:18:48.657 - [Debug] [Init] Auto Access Theshold Achieved: false
2022-10-25 18:18:48.664 - [Info] SSH man-in-the-middle server for 10.0.3.54 listening on 10.0.3.1:1003
2022-10-25 18:18:48.969 - [Debug] [Connection] Attacker connected: 89.121.228.38 | Client Identification: SSH-2.0-Go
2022-10-25 18:18:49.087 - [Debug] [Connection] Attacker connected: 117.80.212.33 | Client Identification: SSH-2.0-Go
2022-10-25 18:18:49.121 - [Debug] [Connection] Attacker connected: 219.140.162.82 | Client Identification: SSH-2.0-Go
2022-10-25 18:18:49.496 - [Debug] [Connection] Attacker connected: 120.236.74.234 | Client Identification: SSH-2.0-Go
2022-10-25 18:18:49.817 - [Debug] [Auth] Attacker 89.121.228.38 trying to authenticate with 'none'
2022-10-25 18:18:49.817 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 18:18:50.126 - [Debug] [Auth] Attacker 89.121.228.38 trying to authenticate with 'publickey'
2022-10-25 18:18:50.128 - [Debug] [Auth] Attacker authentication error: Publickey authentication failed
2022-10-25 18:18:50.128 - [Debug] [Auth] Attacker: 89.121.228.38 has so far made 1 attempts. Remaining: 5 attempts
2022-10-25 18:18:50.379 - [Debug] [Auth] Attacker 117.80.212.33 trying to authenticate with 'none'
2022-10-25 18:18:50.380 - [Debug] [Auth] Attacker authentication error: No authentication method provided
2022-10-25 18:18:50.437 - [Debug] [Connection] Attacker closed the connection
2022-10-25 18:18:50.584 - [Debug] [Connection] Attacker connected: 89.121.228.38 | Client Identification: SSH-2.0-Go
2022-10-25 18:18:50.840 - [Debug] [Auth] Attacker 117.80.212.33 trying to authenticate with 'password'
2022-10-25 18:18:50.840 - [Debug] [Auto Access] Attacker: 117.80.212.33, Threshold: 1, Attempts: 1
2022-10-25 18:18:50.840 - [Debug] [Auto Access] Compromising the honeypot
2022-10-25 18:18:50.840 - [Debug] [Auto Access] Adding the following credentials: 'root:1'
